auto-discovery/kubernetes/controllers/container_scan_controller.go

```diff
@@ -143,13 +143,17 @@ func cleanupImageID(imageID string, log logr.Logger) string {
 }
 
 func getScanName(imageID string) string {
+	//function builds string like: scan-_appName_-at-_imageID_HASH_ eg: scan-nginx-at-0123456789
 	baseScanName := "scan-"
```

Comment:
Thinking about it, this results in some "weird" scan names like: "scan-scan-nmap-xxxyz". Might be worth to change this in another PR to something like "auto-scan-..." to clarify the relation to the autodiscovery.

Reply:
Sadly this would be a breaking change, so we have to wait for the next braking release. See https://github.com/secureCodeBox/secureCodeBox/issues/1193